public/app/features/dashboard/utils/tracking.test.ts

```diff
@@ -30,10 +35,16 @@ describe('trackDashboardLoaded', () => {
       uid: 'dashboard-123',
       title: 'Test Dashboard',
       schemaVersion: model.schemaVersion, // This value is based on public/app/features/dashboard/state/DashboardMigrator.ts#L81
-      panels_count: 2,
+      panels_count: 7,
       variable_type_query_count: 2,
       variable_type_interval_count: 1,
       version_before_migration: 16,
+      panel_type_row_count: 1,
+      panel_type_undefined_count: 1,
```

Comment:
It seems [this one](https://github.com/grafana/grafana/pull/79421/files#diff-7de242c003cd5fd60e6bcc0ddc6a86623fd42d23610bb22ef5e86c5454c852d2R14) is missing the type 😄